interface/patient_tracker/patient_tracker.php

```diff
@@ -215,26 +242,14 @@ function openNewTopWindow(newpid,newencounterid) {
  </tr>
 
 <?php
-#define variables, future enhancement allow changing the to_date and from_date 
-#to allow picking a date to review
- 
-$appointments = array();
-$from_date = date("Y-m-d");
-$to_date = date("Y-m-d");
-$datetime = date("Y-m-d H:i:s");
-
-# go get the information and process it
-$appointments = fetch_Patient_Tracker_Events($from_date, $to_date);
-$appointments = sortAppointments( $appointments, 'time' );
-
 	foreach ( $appointments as $appointment ) {
 
                 # Collect appt date and set up squashed date for use below
                 $date_appt = $appointment['pc_eventDate'];
                 $date_squash = str_replace("-","",$date_appt);
 
                 # Collect variables and do some processing
-                $docname  = $appointment['ulname'] . ', ' . $appointment['ufname'] . ' ' . $appointment['umname'];
+                $docname  = $chk_prov['uprovider_id'];
```

Comment:
Actually, I am still confused here.
1. $chk_prov['uprovider_id'] is an array and this shouldn't work.
2. This will break also if there is more than 1 provider

Am I missing something?